src/lib/Server/Output/ValueObjectVisitor/Role.php

```diff
@@ -20,20 +22,22 @@ class Role extends ValueObjectVisitor
     /**
      * Visit struct returned by controllers.
      *
-     * @param \Ibexa\Contracts\Rest\Output\Visitor $visitor
-     * @param \Ibexa\Contracts\Rest\Output\Generator $generator
-     * @param Role|\Ibexa\Contracts\Core\Repository\Values\User\RoleDraft $data
+     * @param \Ibexa\Contracts\Core\Repository\Values\User\Role|\Ibexa\Contracts\Core\Repository\Values\User\RoleDraft $data
```

Comment:
`RoleDraft` extends `Role` without any extra fields or methods, so it can be simply a `Role`.
Can't add suggestion here for some reason:
```php
     * @param \Ibexa\Contracts\Core\Repository\Values\User\Role $data
```